lib/src/barback/dartdevc/dartdevc_environment.dart

```diff
@@ -0,0 +1,266 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:barback/barback.dart';
+import 'package:cli_util/cli_util.dart' as cli_util;
+import 'package:path/path.dart' as p;
+
+import 'dartdevc.dart';
+import 'module.dart';
+import 'module_computer.dart';
+import 'module_reader.dart';
+import 'scratch_space.dart';
+import 'summaries.dart';
+
+import '../../io.dart';
+import '../../log.dart' as log;
+import '../../package_graph.dart';
```

Comment:
Tiny nit. I think elsewhere in pub, we put relative imports that start with "../" in the same section as but above other relative imports.

I'm not picky either way, but it would be good to be consistent.